packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -688,6 +696,13 @@ export class ApexDebug extends LoggingDebugSession {
           typeof isvDebuggerUrl === 'undefined'
         ) {
           response.message = nls.localize('invalid_isv_project_config');
+          // telemetry for the case where the org-isv-debugger-sid and/or org-isv-debugger-url config variable is not set
+          this.sendEvent(
+            new Event(SEND_METRIC_EVENT, {
+              subject: nls.localize('invalid_isv_project_config'),
+              type: 'startIsvDebuggerConfigError'
+            } as Metric)
```

Comment:
The "as" assertions are unnecessary, please remove them in all locations with this same usage pattern